mct/m_SPMDutils.F90

```diff
@@ -24,6 +24,16 @@
 !
 ! !INTERFACE:
 
+! Code added as a work around for poor performance or incorrect functionality
+! in MPI libraries
+!
+#ifndef _USE_MPI_RSEND
```

Comment:
@worleyph take a look at mct/mpeu/m_FcComms.F90.  It has a similar logic but RSEND is left ON by default unless one defines _NO_MPI_RSEND.  Want to change it to be the same as this?